packages/framer-motion/src/frameloop/context.ts

```diff
@@ -0,0 +1,7 @@
+export interface FrameloopContext {
+    current: void | VoidFunction[]
+}
+
+export const frameloopContext: FrameloopContext = {
+    current: undefined,
+}
```

Comment:
Is there a more descriptive name for this than frameloop context? I'm wondering if we can avoid the name "context" as that's already a react concept.

Reply:
Potentially? But I don't think React has monopoly on "context", this is also a JS concept and likewise we would probably feel ok calling something "state"

Reply:
Yeah, _if_ we can come up with something better, that might be great. I agree that React doesn't have a monopoly, but this just is a global variable, so we might as well name it to what it contains, right?